src/QueryLogger.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Muffin\Webservice;
+
+class QueryLogger extends \Cake\Database\Log\QueryLogger
```

Comment:
@markstory @lorenzo would it make sense to move the ```\Cake\Database\Log\QueryLogger``` to ```\Cake\Datasource\Log\QueryLogger``` so that it its not 'database' specific?